data-document-processor/build.gradle

```diff
@@ -16,8 +16,9 @@ dependencies {
 	testImplementation mn.micronaut.inject.java.test
 	testImplementation projects.dataDocumentTck
 	testImplementation projects.dataMongodb
+	testImplementation projects.dataAzureCosmos
```

Comment:
Please move the test to the Cosmos module

Reply:
Done in https://github.com/micronaut-projects/micronaut-data/pull/1760/commits/bdd349d0f21d92d93b19faa717443e6167c91e7a